.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python: 2.7
-install: "pip install flake8"
-script: "flake8 ."
+install:
+  - pip install flake8
+script:
+  - flake8 .
```

Comment:
We should run flake8 via Tox, see https://github.com/mozilla/mozillians-tests/blob/master/.travis.yml for an example.